107/src/test/resources/ehcache-107-serializer.xml

```diff
@@ -12,13 +12,13 @@
     </ehcache:default-serializers>
   </ehcache:service>
 
-  <cache alias="cache" storeByValueOnHeap="true">
+  <cache alias="cache">
     <key-type>java.lang.Long</key-type>
     <value-type>java.lang.String</value-type>
     <heap size="20" unit="entries"/>
   </cache>
 
-  <cache alias="cache1" storeByValueOnHeap="true">
+  <cache alias="cache1">
```

Comment:
With your changes, the serialisers are not used. Can you see with @rishabhmonga what was the objective of this test and adapt / remove?